core/trino-main/src/main/java/io/trino/server/security/ServerSecurityModule.java

```diff
@@ -73,7 +73,10 @@ protected void setup(Binder binder)
             configBinder(certificateBinder).bindConfig(CertificateConfig.class);
```

Comment:
squash this commit with `Bind PasswordAuthenticatorManager using OptionalBinder `